airflow/providers/google/cloud/hooks/automl.py

```diff
@@ -1,642 +0,0 @@
-#
```

Comment:
Technically speaking - we didn't actually deprecate this hook, but as GCP disables their API it will become unusable anyway.
Should we keep it with an exception raised on initialization and delete it later?